kubernetes/structure_network_policy.go

```diff
@@ -214,7 +203,7 @@ func expandNetworkPolicyV1Ports(l []interface{}) (*[]networkingv1.NetworkPolicyP
 			policyPorts[i].Protocol = &v
 		}
 	}
-	return &policyPorts, nil
+	return policyPorts
```

Comment:
nit: Changing the function to also no longer return a pointer could/should have been a separate PR

Reply:
Reverted this change. Thanks!